internal/cli/runner_inspect.go

```diff
@@ -0,0 +1,136 @@
+package cli
+
+import (
+	"fmt"
+	"github.com/dustin/go-humanize"
+	"github.com/golang/protobuf/ptypes"
+	"reflect"
+	"strings"
+
+	"github.com/golang/protobuf/jsonpb"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
+	"github.com/hashicorp/waypoint/internal/clierrors"
+	"github.com/hashicorp/waypoint/internal/pkg/flag"
+	pb "github.com/hashicorp/waypoint/pkg/server/gen"
+	"github.com/posener/complete"
+)
+
+type RunnerInspectCommand struct {
+	*baseCommand
+
+	flagJson bool
+}
+
+func (c *RunnerInspectCommand) Run(args []string) int {
+	// Initialize. If we fail, we just exit since Init handles the UI.
+	if err := c.Init(
+		WithArgs(args),
+		WithFlags(c.Flags()),
+		WithNoLocalServer(), // no auth in local mode
+		WithNoConfig(),
+	); err != nil {
+		return 1
+	}
+
+	if len(c.args) == 0 {
+		c.ui.Output("Runner name required.", terminal.WithErrorStyle())
+		return 1
+	}
+	id := c.args[0]
+
+	resp, err := c.project.Client().GetRunner(c.Ctx, &pb.GetRunnerRequest{RunnerId: id})
+	if err != nil && status.Code(err) != codes.NotFound {
```

Comment:
Do we need another error case for when a requested runner profile is not found? I think it'll ignore that case here.

Reply:
It panics:

```
[go][main](4)$ waypoint runner inspect 01FSW91WAMHBQZ542NKKR2ST1Q

» Runner:
              ID: 01FSW91WAMHBQZ542NKKR2ST1Q
  Adoption State: new
            Kind: remote
          Labels: map[]
          Online: false
          
[go][main](4)$ waypoint runner inspect puppy

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0x34dac60]

goroutine 1 [running]:
github.com/hashicorp/waypoint/internal/cli.(*RunnerInspectCommand).Run(0xc000efb2e0, {0xc000eaa570, 0xc5, 0x3e0eeb0})
        /Users/clint/go-src/github.com/hashicorp/waypoint/internal/cli/runner_inspect.go:69 +0x580
github.com/mitchellh/cli.(*CLI).Run(0xc000dd0000)
        /Users/clint/Go/pkg/mod/github.com/mitchellh/cli@v1.1.2/cli.go:262 +0x5f8
github.com/hashicorp/waypoint/internal/cli.Main({0xc00006e080, 0x3717960, 0xc0000001a0})
        /Users/clint/go-src/github.com/hashicorp/waypoint/internal/cli/main.go:127 +0x545
main.main()
        /Users/clint/go-src/github.com/hashicorp/waypoint/cmd/waypoint/main.go:14 +0x7f
```